### ISSUE TITLE
IRL input Support

### ISSUE THREAD

**Issue by haywirephoenix** [Unity-Technologies/EditorXR#570]
EditorXR could impove the way that we interact with Unity even outside of VR.

- Enable EXRs window to be viewed outside of VR 
- Add Kb+Mouse / Gamepad support to control the camera and transform/place objects
- Exr cam can work as a free-camera in game and can optionally save changes in play mode.

**Comment by amirebrahimi** [comment]
Thanks for the suggestion. I think @mtschoen-unity has written a utility like this at some point for testing purposes, but it hasn't made it into the repo yet.

**Comment by mtschoen-unity** [comment]
Indeed! We're working on a way to flow touch/click input through the input module so that you can interact with EditorXR using more than just VR controllers. We're not quite ready to share any of this but we're heading in this direction for sure.
